pkg/routeagent_driver/handlers/ovn/uninstall.go

```diff
@@ -41,11 +43,15 @@ func (ovn *Handler) Stop(uninstall bool) error {
 		klog.Errorf("Error deleting Submariner bridge %q due to %v", ovnK8sSubmarinerBridge, err)
 	}
 
-	if ovn.isGateway {
-		err = ovn.cleanupGatewayDataplane()
-		if err != nil {
-			klog.Errorf("Error cleaning the gateway routes to %v", err)
-		}
+	err = ovn.cleanupRoutes()
```

Comment:
Everything looks good to me, I have one small comment about the name of the function (I'm not good at choosing names :-) ), maybe
'CleanupIpRules' is more appropriate here?

Reply:
IMHO routes will be easier for many people to co-relate compared to ipRules.